src/xcode/ENA/ENA/Source/Scenes/Settings/BackgroundAppRefresh/InfoBoxView.swift

```diff
@@ -0,0 +1,182 @@
+		settingsButton.setTitle(viewModel.settingsText, for: .normal)
+		
+		settingsAction = viewModel.settingsAction
+		shareAction = viewModel.shareAction
+
+		instructionsStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+		nonAccessibilityConstraints.removeAll()
+		accessibilityConstraints.removeAll()
+
+		for instruction in viewModel.instructions {
+			let titleLabel = ENALabel()
+			titleLabel.text = instruction.title
+			titleLabel.numberOfLines = 0
+			titleLabel.style = .headline
+
+			instructionsStackView.addArrangedSubview(titleLabel)
+
+			for index in 0..<instruction.steps.count {
+				instructionsStackView.addArrangedSubview(view(for: instruction.steps[index], index: index))
+			}
+		}
+
+		updateConstraintsForCurrentTraitCollection()
+	}
+	
+	// MARK: - Private
+
+	@IBOutlet private weak var infoBoxTitle: ENALabel!
+	@IBOutlet private weak var infoBoxText: ENALabel!
+	@IBOutlet private weak var instructionsStackView: UIStackView!
+	@IBOutlet private weak var settingsButton: UIButton!
+	@IBOutlet private weak var shareButton: UIButton!
+
+	private var nonAccessibilityConstraints = [NSLayoutConstraint]()
+	private var accessibilityConstraints = [NSLayoutConstraint]()
+	
+	private var shareAction: () -> Void = { }
+	private var settingsAction: () -> Void = { }
+	
+	private func loadViewFromNib() {
+		let nib = UINib(nibName: "InfoBoxView", bundle: nil)
+		guard let view = nib.instantiate(withOwner: self, options: nil).first as? UIView else {
+			return
+		}
+
+		addSubview(view)
+		view.translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			view.topAnchor.constraint(equalTo: topAnchor),
+			view.trailingAnchor.constraint(equalTo: trailingAnchor),
+			view.bottomAnchor.constraint(equalTo: bottomAnchor),
+			view.leadingAnchor.constraint(equalTo: leadingAnchor)
+		])
+	}
+	
+	private func view(for step: InfoBoxViewModel.InstructionStep, index: Int) -> UIView {
+		let containerView = UIView()
+		containerView.isAccessibilityElement = true
+		containerView.accessibilityLabel = "\(index + 1). \(step.text)"
+
+		let iconImageView = UIImageView(image: step.icon)
+
+		containerView.addSubview(iconImageView)
+		iconImageView.translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			iconImageView.widthAnchor.constraint(equalToConstant: 28),
+			iconImageView.heightAnchor.constraint(equalToConstant: 28),
+			iconImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+			iconImageView.topAnchor.constraint(greaterThanOrEqualTo: containerView.topAnchor)
+		])
+
+		let enumerationLabel = ENALabel()
+		enumerationLabel.text = "\(index + 1)."
+		enumerationLabel.numberOfLines = 1
+		enumerationLabel.style = .headline
+		enumerationLabel.setContentCompressionResistancePriority(.required, for: .horizontal)
+
+		containerView.addSubview(enumerationLabel)
+		enumerationLabel.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Doesn't it make more sense to set `translatesAutoresizingMaskIntoConstraints` to `false` before adding it to `containerView`?

Reply:
Same for the other views that are added prior to setting `translatesAutoresizingMaskIntoConstraints` to `false`.

Reply:
Changed.

TBH: Didn't knew that the order makes a difference :)